test/end-to-end/helpers/GeneralHelpers.js

```diff
@@ -0,0 +1,19 @@
+function decodeHTML(str) {
+    return str.replace(/&([a-zA-Z]+);/g,
```

Comment:
I tried the decodeURI and decodeURIComponent on the Welsh chars and it didn't seem to produce the expected result for me. This has proven to work well in all the different browsers, so it might be worth looking at in a separate PR as a future improvement?